core/server/api_container/server/startosis_engine/startosis_packages/package_content_provider.go

```diff
@@ -29,10 +29,14 @@ type PackageContentProvider interface {
 	// ClonePackage clones the package with the given id and returns the absolute path on disk
 	ClonePackage(packageId string) (string, *startosis_errors.InterpretationError)
 
-	// GetAbsoluteLocatorForRelativeModuleLocator returns the absolute package path for a relative module path and replace the package path if
+	// GetAbsoluteLocatorForRelativeLocator returns the absolute package path for a relative module path and replace the package path if
```

Comment:
does the `module`  in comment still apply; maybe its just files!

Reply:
yes, files is more accurated, so I'll update the comment